src/utils/renderHelpers.ts

```diff
@@ -45,17 +45,20 @@ export function findFirstOccurrenceOutsideComment(
 }
 
 function findFirstLineBreakOutsideComment(code: string, start: number = 0) {
-	let commentStart, lineBreakPos;
+	let commentStart, lineBreakPos, nextChar;
 	while (true) {
-		commentStart = code.indexOf('/*', start);
 		lineBreakPos = code.indexOf('\n', start);
-		if (commentStart === -1) break;
-		if (lineBreakPos >= commentStart) {
-			lineBreakPos = -1;
-		} else if (lineBreakPos !== -1) break;
+		do {
+			commentStart = code.indexOf('/', start);
+			if (commentStart === -1 || commentStart > lineBreakPos) return lineBreakPos;
+			nextChar = code[commentStart + 1];
```

Comment:
charCodeAt perhaps?

Reply:
Of course! Has been changed. I also slightly improved it to not misinterpret `/*/` as an immediately closed block comment